pkg/local_object_storage/metabase/util.go

```diff
@@ -4,13 +4,17 @@ import (
 	"bytes"
 	"crypto/sha256"
 	"fmt"
+	"math/big"
 
 	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
 	"github.com/nspcc-dev/neofs-sdk-go/object"
 	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
 	"go.etcd.io/bbolt"
 )
 
+// UTF-8 data separator used in DB.
+var utf8Delimiter = []byte("\xc0\x80") // U+0000
```

Comment:
Have you tried `[]byte{0}`?

Reply:
zero byte is a utf8 char, so we can miss sep b/w key and val after their concat

Reply:
I thought it's an invalid one.

Reply:
do u think we need LV format, or this one is OK?

Reply:
lets start with 0xFF - shorter, and cannot be a part of attribute